pkg/apis/multicluster/v1alpha1/gameserverallocationpolicy.go

```diff
@@ -54,3 +57,124 @@ type GameServerAllocationPolicyList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []GameServerAllocationPolicy `json:"items"`
 }
+
+// clusterToPolicyMap map type definition for cluster to policy map
+type clusterToPolicyMap map[string][]*GameServerAllocationPolicy
```

Comment:
while commenting on style, don't need the "Map" suffix here.

https://dave.cheney.net/2019/01/29/you-shouldnt-name-your-variables-after-their-types-for-the-same-reason-you-wouldnt-name-your-pets-dog-or-cat

Reply:
Done. Thanks!